package.json

```diff
@@ -13,8 +13,6 @@
   "scripts": {
     "build": "gulp build",
     "start": "http-server ./demo",
-    "gulpversion": "gulp build && git add .",
-    "prepublish": "typings install && gulp build",
```

Comment:
This makes it impossible to install the package locally from package.json without typings globally installed.

Reply:
Why? It's just omitting prepublish step. users do not publish packages so they do not run this step.